packages/cna-template/template/frameworks/jest/jest.config.js

```diff
@@ -2,7 +2,12 @@ module.exports = {
   moduleNameMapper: {
     '^@/(.*)$': '<rootDir>/$1',
     '^~/(.*)$': '<rootDir>/$1',
-    '^vue$': 'vue/dist/vue.common.js'
+    '^vue$': 'vue/dist/vue.common.js',
+    <%_ if (composition) { _%>
+    '@nuxtjs/composition-api': '@nuxtjs/composition-api/lib/entrypoint.js',
+    // alternatively, depending on your node version
+    // '@nuxtjs/composition-api': '@nuxtjs/composition-api/entrypoint',
```

Comment:
I would recommend defaulting to this one as newer node versions will only recognise this format

Reply:
I think the first should work as we have `"./": "./"` in `exports` of @nuxtjs/composition-api ?

BTW, trailing "/" suffix for subpath will be removed in a future Node.js release.

Reply:
Good point! Yes, but the trailing `/` is deprecated in favour of `/*`.